src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private int maxNum = 100;
```

Comment:
Let's create it as a constant by naming conventions also. See [here](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names).